nuxt.config.js

```diff
@@ -159,6 +159,16 @@ export default async () => {
             historyTitle: "更新履歴",
             opendataTitle: "オープンデータ・API (β)",
           },
+          "zh-CN": {
+            siteTitle: "Genshin English Dictionary",
+            indexTitleDesc: "an online English-Chinese dictionary for terms in Genshin Impact",
+            wordIdTitle: "\"{zhCN}\" is \"{en}\" in English",
+            wordIdDescription: "\"{zhCN}\" is \"{en}\" in English (and \"{ja}\" in Japanese).",
```

Comment:
I think we should add them, but the text string length is a problem.

Maybe we can use an abbreviation, for example:

`wordIdTitle: "\"{zhCN}\": \"{en}\"（英）、\"{ja}\"（日）",`

Reply:
I'm not sure how Baidu and other Chinese search engines behaves, but `wordIdTitle: "\"{zhCN}\": \"{en}\"（英）、\"{ja}\"（日）",` might affect SEO.
If the user searches by "原神 英语", the search engine might give us the lower rank. I guess the word "英语" (or equivalent) should be included.

It would be an option to give up including translations in the title.
e.g. `wordIdTitle: "English and Japanese translation for \"{zhCN}\"",` (in Chinese)

Reply:
Maybe your advice is a proper choice. I will translate them like it.